sycl/plugins/unified_runtime/ur/adapters/native_cpu/command_buffer.cpp

```diff
@@ -0,0 +1,131 @@
+//===--------- command_buffer.cpp - NativeCPU Adapter ---------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===-----------------------------------------------------------------===//
+
+#include "common.hpp"
+
+/// Stub implementations of UR experimental feature command-buffers
+/// Taken almost unchanged from another adapter. Perhaps going forward
+/// these stubs could be defined in core UR as the default which would
```

Comment:
possibly, but that would imply having C++ headers for the definition of the UR interfaces, to allow adapters to have overrides. Each adapter needs to implement the stubs with their internal logic. But maybe in the future, some of the logic for command buffers could have a shared part, common to all adapters, and then other parts that are adapter-specific.